NSS-Day-Cohort-53/giffygram-spazzy-kermits
# Sh user profile

#### Changes Made
1. Added file `profile.js` to `postFeed` directory.
2. Created a profile page
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout sh-userProfile
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > Profile should display once a user  is clicked, profile will display user first and last name, post count, and al messages sent to and from the chosen user and the current logged in user
    > Example: when user clicks a person (eventually) profile will display
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
